app/jobs/set_governing_apo_job.rb

```diff
@@ -57,4 +62,22 @@ def update_druid_count
     bulk_action.update(druid_count_total: pids.length)
     bulk_action.save
   end
+
+  def open_new_version(object, log)
+    if DorObjectWorkflowStatus.new(object.pid).can_open_version?
+      begin
+        vers_md_upd_info = {
+          :significance => 'minor',
+          :description => 'Set new governing APO',
+          :opening_user_name => current_user.to_s
+        }
+        object.open_new_version({:vers_md_upd_info => vers_md_upd_info})
+      rescue Dor::Exception => e
+        log.puts("#{Time.current} Failed to open new version for #{object.pid} (bulk_action.id=#{bulk_action.id}): #{e}")
+        return
```

Comment:
plus, wouldn't this swallow the exception and let it proceed to try setting the APO?  (running the code in my head and not in the app, so very possible i'm guessing wrong 😄 )